src/Service/DoctrineObjectHydratorFactory.php

```diff
@@ -6,7 +6,7 @@
 
 use Doctrine\Laminas\Hydrator\DoctrineObject;
 use Doctrine\ODM\MongoDB\DocumentManager;
-use Psr\Container\ContainerInterface;
+use Interop\Container\ContainerInterface;
```

Comment:
What's the benefit of this change?

> Starting Feb. 13th 2017, container-interop is officially deprecated in favor of PSR-11.

Reply:
The [FactoryInterface](https://github.com/laminas/laminas-servicemanager/blob/3.11.x/src/Factory/FactoryInterface.php) from the latest Laminas ServiceManager still uses `Interop\Container\ContainerInterface`. Hence, whenever implementing the Laminas interfaces, we are using Interop's ContainerInterface.

I wanted to keep this consistent so that we only need to require one of `container-interop/container-interop`  and `psr/container`. Otherwise, we'd need to require both.